test/fixtures/dialects/spark3/select_regular_identifier_without_leading_digits.sql

```diff
@@ -0,0 +1,2 @@
+select 1 as plain_identifier_with_no_tricks_1;
```

Comment:
Surprisingly, plain identifiers in spark can start with digits like `0_is_a_legal_identifier`, and these also fail to parse. However, since they parse when delimited and making that worked seemed like it would be a bigger impact than just fixing the quote character, I stopped investigating.

Reply:
this should parse according to the definition in ANSI, but I can see an issue with the current regex so I'll look at fixing this. Can you remove this file from the PR as it's not relevant to the back tick identifiers?

Reply:
The leading numeric issue will be fixed by #2114 so thanks for highlighting this 👍